registration/serializers.py

```diff
@@ -109,6 +109,47 @@ def create(self, validated_data):
 
         return family
 
+    def update(self, instance, validated_data):
+        validated_data.pop("current_enrolment")  # Return value needed for future ticket
```

Comment:
what future ticket is this referring to?

Reply:
didn't actually check the board, but I'm assuming there will be a task to allow updates to current enrolment? Could be wrong, but need to pop current enrolment anyway before I update the fields on family.

Reply:
it's already done! was in this PR ([#77](https://github.com/uwblueprint/project-read-frontend/pull/77))

sounds good about popping it - we can remove the comment 😄